src/map/lua/lua_baseentity.cpp

```diff
@@ -12419,6 +12419,31 @@ inline int32 CLuaBaseEntity::addFullGambit(lua_State* L)
     return 0;
 }
 
+inline int32 CLuaBaseEntity::trustPartyMessage(lua_State* L)
+{
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_TRUST);
+    TPZ_DEBUG_BREAK_IF(lua_isnil(L, 1) || !lua_isnumber(L, 1));
+
+    auto PTrust = static_cast<CTrustEntity*>(m_PBaseEntity);
+
+    auto p0 = lua_tointeger(L, 1);
+    auto p1 = 0;
+    auto p2 = 711;
```

Comment:
Uncertain how I feel about the assignment here instead of using the ints directly in the creation of the new packet.

Are p1 and p2 going to be capable of being changed later?

Reply:
Yeah, they're magic numbers specific to this usage, so I'll move them down 👍